### DISCUSSION TITLE
Thank You!!!!!!!!!!!!!!!!!!!!!!!!!!!!    ⭐⭐⭐⭐⭐

### DISCUSSION THREAD

**Discussion by meyerchak** [nextcloud/cookbook#1189] [Feedback]
This software is amazing.  Thank you so much for developing a private self-hosted FOSS CookBook.  I am a big fan and i think CookBook is a Nextcloud must have!  The UX/UI is great on desktop and android.  the android app is on my Samshit S10e phone's home screen.  i use it a lot.

Thank you to all the developers!

Bravo!

Cheers

**Comment by christianlupus** [comment]
Thank you very much for your kind feedback! It is good to hear that you are satisfied with the cookbook app.

Although, the Android app is a third party app. There are multiple ones available. I would like to forward to the appropriate developer your feedback, but I do not know who to mention.

**Reply by meyerchak** [reply]
Nextcloud CookBook is a must have app, should be a feature addon.  Can NC CookBook be entered in the Nextcloud app contest for the 2022 developer conference?  CookBook would have my vote!

I posted a thank you message for Teifun2 in the repo.  

Once again great work and thank you to the whole team!

**Reply by christianlupus** [reply]
Unfortunately, we cannot participate in the contest as this is reserved to newly created apps. Sorry but thanks for your offer to support.
